torchvision/models/video/resnet.py

```diff
@@ -189,9 +211,15 @@ def __init__(self):
 
 class VideoResNet(nn.Module):
 
-    def __init__(self, block, conv_makers, layers,
-                 stem, num_classes=400,
-                 zero_init_residual=False):
+    def __init__(
+        self,
+        block: Type[Union[BasicBlock, Bottleneck]],
```

Comment:
I believe block should be an `nn.Module`. The intention is to allow users pass their own custom blocks

Reply:
Alright, but apart from `nn.Module` default class, it needs an `expansion` attribute

Reply:
You are right, leave as is. :)